scargo/file_generators/templates/conan/profile.j2

```diff
@@ -25,7 +37,7 @@ compiler.cppstd={{ config.project.cxxstandard }}
 os_build=Linux
 arch_build=x86_64
 
-build_type={{profile}}
+build_type={{ profile if (profile in standard_profiles) else config.profiles[profile].build_type }}
```

Comment:
Here we had code smell... 
Basically backward compatybility of toml enforce strange code here. 
We need TODO: notices why it look so strange. And in version 2.0.0 of scargo this hack had to be removed and all profiles in toml had to be connected with conan profile not the build type. and we need to enforce definition of build_type there.

Reply:
Comment with TODO has been added